message_sender/tests.py

```diff
@@ -605,6 +605,34 @@ def test_event_delivery_failed(self):
         self.assertEquals(False, self.check_logs(
             "Message: 'Simple outbound message' sent to '+27123'"))
 
+    def test_create_inbound_data(self):
```

Comment:
Maybe we should mention something about Junebug in this test name/docstring? Just to show that we're testing the Junebug -> vumi message format translation.